data/aecore/.minerva/accounts_uat.json

```diff
@@ -0,0 +1,4 @@
+{
+   "ak_2a1j2Mk9YSmC1gioUq4PWRm3bsv887MbuRVwyv4KaUGoR1eiKi": 100000000000002
```

Comment:
I see this in testnet:
http://52.10.46.160:3013/v2/accounts/ak_2a1j2Mk9YSmC1gioUq4PWRm3bsv887MbuRVwyv4KaUGoR1eiKi
balance 89845016266259

Reply:
Enhancement proposed in https://github.com/aeternity/aeternity/pull/2171